src/Raven.Server/Documents/ETL/Providers/SQL/RelationalWriters/DbProviderFactories.cs

```diff
@@ -38,7 +38,7 @@ public static DbCommandBuilder InitializeCommandBuilder(this DbProviderFactory f
                     Start = "[",
                     End = "]"
                 };
-            if (factory is NpgsqlFactory)
+            if (factory is NpgsqlFactory || factory is OracleClientFactory)
```

Comment:
Is it a breaking change? Why we never run into this before?

Reply:
no, its used for proper SQL script created by our ETL, I think there are no tests for oracle.